<launch>
    <arg name="weight_file"                 default="$(find dr_spaam_ros)/weight_file/ckpt_jrdb_pl_dr_spaam_phce_mixup_e20.pth"/>
    <arg name="detector_model"              default="DR-SPAAM"/>    <!-- DROW3 or DR-SPAAM -->
    <arg name="conf_thresh"                 default="0.9"/>
    <arg name="stride"                      default="1"/>       <!-- use this to skip laser points -->
    <arg name="panoramic_scan"              default="false"/>    <!-- Set to true if the scan covers 360 degree -->

    <group ns = "dr_spaam">
        <node pkg="multiple_sensor_person_tracking" type="dr_spaam_ros_service.py" name="dr_spaam_ros_service" output="screen">
            <param name="weight_file"               value="$(arg weight_file)" />
            <param name="detector_model"            value="$(arg detector_model)" />
            <param name="conf_thresh"               value="$(arg conf_thresh)" />
            <param name="stride"                    value="$(arg stride)" />
            <param name="panoramic_scan"            value="$(arg panoramic_scan)" />
            <rosparam>
                server:
                    scan:
                        service: /dr_spaam_ros_server
            </rosparam>
        </node>
    </group>
</launch>

